### PR TITLE
Update SECURITY.md to support changes made on how how we receive reports from externals about security issues

### PR DESCRIPTION
**Pull request by kgalli**
Changes regarding how we receive reports from externals about security issues in our code (or anything else, like GDPR related incidents, data breaches, security vulnerabilities and more) make it necessary to update the SECURITY.md accordingly.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+We acknowledge that every line of code that we write may potentially contain security issues.
+We are trying to deal with it responsibly and provide patches as quickly as possible.
+
+We host our bug bounty program on HackerOne, it is currently private, therefore if you would like to report a vulnerability and get rewarded for it, please ask to join our program by filling this form:
```
Comment (cberg-zalando) [inline SECURITY.md:4]: s/HackerOne, it/HackerOne. It`

Comment (cberg-zalando) [reply SECURITY.md:4]: Alternatively you can also make the word `form` to be the link.

Comment (kgalli) [reply SECURITY.md:4]: Can you address all change requests here: https://github.com/zalando-incubator/new-project/blob/master/SECURITY.md.

Once it is agreed there we can update here as well. This PR is simply following what is proposed by the sample.

Comment (kgalli) [reply SECURITY.md:4]: That being said I still think we should merge this one and create a new PR to update this Repo once the "master" template got changed accordingly. @cberg-zalando what do you think?

Comment (cberg-zalando) [reply SECURITY.md:4]: Sure. I was not aware that this was based on a project template.

**Comment by kgalli** [comment]
👍

**Comment by cberg-zalando** [comment]
:+1:

**Pull request merged**